dss/api/subscriptions_v1.py

```diff
@@ -173,14 +173,24 @@ def delete(uuid: str, replica: str):
     return jsonify({'timeDeleted': time_deleted}), requests.codes.okay
 
 
-def _unregister_percolate(es_client: Elasticsearch, uuid: str):
-    response = es_client.delete_by_query(index="_all",
-                                         doc_type=ESDocType.query.name,
-                                         body={"query": {"ids": {"type": ESDocType.query.name, "values": [uuid]}}},
-                                         conflicts="proceed",
-                                         refresh=True)
-    if response['failures']:
-        logger.error("Failed to unregister percolate query for subscription %s: %s", uuid, response)
+def _unregister_percolate(es_client: Elasticsearch, subscribed_indexes: List[str], uuid: str):
+    indices = es_client.indices.get('*').keys()
+
+    def chunked(input_list, chunk_size=20):
```

Comment:
was chuck size something arbitrary? or from ES docs?

Reply:
It included everything.